docs/release-1.3/README.md

```diff
@@ -1,6 +1,6 @@
 # Orchestrator Documentation
 
-For comprehensive documentation on the Orchestrator, please visit [https://www.parodos.dev](https://www.parodos.dev).
+For comprehensive documentation on the Orchestrator, please visit [https://www.rhdhorchestrator.io/main/docs/](https://www.rhdhorchestrator.io/main/docs/).
```

Comment:
Hi @bbalakriz , thanks for your PR!
I'd like to suggest this
```suggestion
For comprehensive documentation on the Orchestrator, please visit [https://www.rhdhorchestrator.io](https://www.rhdhorchestrator.io).
```
to don't only guides users to the main documentation but also encourages them to explore additional valuable information available in the blog.